Runtime/CourseController/InitCourseOnSceneLoad.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using Innoactive.Creator.Core;
+using Innoactive.Creator.Core.Configuration;
+using UnityEngine;
+
+namespace Innoactive.Creator.UX
+{
+    /// <summary>
+    /// Initializes the <see cref="CourseRunner"/> with the current selected course on scene start.
+    /// </summary>
+    public class InitCourseOnSceneLoad : MonoBehaviour
+    {
+        private void OnEnable()
+        {
+            InitTraining();
```

Comment:
You can select the language with the menu loaded, is this way here still supporting switching the language?